Aztec/Classes/Public/Extensions/CGRect+Helpers.swift

```diff
@@ -0,0 +1,56 @@
+import CoreGraphics
+
+extension CGRect {
+    /// The position to which a rect is anchored when resized vertically.
+    ///
+    enum VerticalAnchor {
+        case top
+        case middle
+        case bottom
+    }
+
+    /// The position to which a rect is anchored when resized horizontally.
+    ///
+    enum HorizontalAnchor {
+        case left
+        case center
+        case right
+    }
+
+    /// Return a rect resized to a specific size, while anchoring it to one of
+    /// their edges or centers.
+    ///
+    /// By default, changing a CGRect size will resize it anchored to its top
```

Comment:
Yeah, I struggled with the description there. Your description reads easier, but it's also not completely accurate. I wouldn't say that "CGRect does not support anchoring", since it's just a dumb struct. Like you said, setting the size is just an assignment. Let's try another iteration:

> CGRect is typically used in UIKit so the origin is in the upper-left corner and the rectangle extends towards the lower-right corner. This method adds the option to grow or shrink the rect   anchoring to other points.

BTW, I added the UIKit mention since I read the CGRect documentation and found this:

> In the default Core Graphics coordinate space, the origin is located in the lower-left corner of the rectangle and the rectangle extends towards the upper-right corner. If the context has a flipped-coordinate space—often the case on iOS—the origin is in the upper-left corner and the rectangle extends towards the lower-right corner.

To be completely accurate, we should ditch the left/right top/bottom enum values and be context agnostic, but I think we can let it go for this project. Maybe just add a note that the method assumes a upper-left-origin coordinate system.

Thoughts?